.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,43 @@
+name: Benchmark PR vs main
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [ main ]
+  
+jobs:
+  benchmark-delta-linux:
+    runs-on:
+      - runs-on=${{ github.run_id }}
+      - runner=2cpu-linux-x64
+    container: swift:jammy
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: jemalloc dependency
+        run: apt-get update && apt-get install -y libjemalloc-dev
+      - name: Fix Git config
+        run: |
+          git config --global --add safe.directory "${GITHUB_WORKSPACE}"
+      - name: Run benchmarks for PR branch
+        continue-on-error: true
+        run: |
+          swift package -c release --package-path Benchmarks --disable-sandbox benchmark baseline update pull_request
+      - name: Run benchmarks for 'main' branch
+        run: |
+          git stash
+          git checkout main
+          swift package -c release --package-path Benchmarks --disable-sandbox benchmark baseline update main
```

Comment:
We should use static thresholds like SwiftNIO does.
Joakim recently introduced a new command for that as well.

Reply:
Though static thresholds are only supported for p90 i think?
That's also what SwiftNIO is fine with.